cmd/cgr-engine/cgr-engine.go

```diff
@@ -1605,6 +1604,22 @@ func main() {
 			fmt.Println(err.Error())
 			return
 		}
+	} else if cfg.SessionSCfg().Enabled {
```

Comment:
please keep the connection within the previous if, only return/warning if SessionS.Enabled